src/Opulence/Framework/Console/Commands/RenameAppCommand.php

```diff
@@ -57,8 +57,11 @@ protected function define()
                 'newName',
                 ArgumentTypes::REQUIRED,
                 'The new application name'
-            ))
-            ->setHelpText('Be sure to use the correct capitalization for your app names.');
+            ));
+
+        if ($this instanceof Command) {
```

Comment:
Actually it's in a fair bit of mess at the moment. Problem is, `addArgument` promises to return an `ICommand`, but `setHelpText` is not defined on `ICommand`. (Not the only cases where this could bite back.)